### DISCUSSION TITLE
How to add new language to CodeMirrorPlugin

### DISCUSSION THREAD

**Discussion by nekronomekron** [mdx-editor/editor#591] [Q&A]
I have the editor up and running, code blocks are working but now I want to add a new language to the CodeMirror plugin.
I also have the `LanguageSupport` object but I don't know how to forward this object to CodeMirror.

**Comment by petyosi** [comment]
The plugin accepts a `codeMirrorExtensions` config param. https://mdxeditor.dev/editor/api/functions/codeMirrorPlugin

**Comment by nekronomekron** [comment]
I've seen this property but for some reason, I can't get it to work.
According to the documentation of CodeMirror I must wrap the LanguageSupport object into a Compartment, which I did. But as soon as I insert the Compartment to the CodeMirror plugin, none of the syntax highlighting is working anymore (text, css, js, ..).

I also added the "basicSetup" extension from CodeMirror, with the same results.
